css/index.css

```diff
@@ -52,8 +52,43 @@ table {
 }
 
 html, body {
-    height: 100%;
+    width: 1000px;
     font-family: 'Roboto', sans-serif;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 10px auto;
+    border: 2px solid black;
+}
+
+/* NEW CODE HERE ***************************/
+.header {
+    display: flex;
+    justify-content: space-between;
+    padding: 0px 10%;
+}
+
+#LambdaLOGO {
+}
```

Comment:
When you are finished with an assignment it's always a good habit to go back and clean up your code. For example, `#LambdaLOGO` isn't needed here.